### PR TITLE
Minor wording change in timeseries module's README

### PR DESCRIPTION
**Pull request by terrytangyuan**

### CODE REVIEW

File: tensorflow/contrib/timeseries/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 TensorFlow Time Series (TFTS) is a collection of ready-to-use classic models
 (state space, autoregressive), and flexible infrastructure for building
-high-performance time series models whatever the architecture. It includes tools
+high-performance time series models with custom model architecture. It includes tools
```
Comment (allenlavoie) [inline tensorflow/contrib/timeseries/README.md]: "with custom architectures" (add an "s", remove redundant "model")?

**Review by allenlavoie — CHANGES_REQUESTED** [review]
Thanks! Some minor grammar nitpicks.

**Review by drpngx — APPROVED** [review]
Addressed Allen's comments.

**Comment by terrytangyuan** [comment]
Thanks for fixing it!

**Review by allenlavoie — APPROVED** [review]

**Pull request merged**